codespell_lib/data/dictionary.txt

```diff
@@ -31364,6 +31455,7 @@ shreak->shriek
 shreshold->threshold
 shriks->shrinks
 shrinked->shrunk, shrank,
+shs->ssh
```

Comment:
Could be the NHS in the UK. Not sure what to do here.

Reply:
Let's suggest both. NHS makes sense.